src/modules/user/coupon/OrderCoupon.tsx

```diff
@@ -0,0 +1,102 @@
+import { H5, Text } from "@/modules/common/components/Typography";
+import { coupons } from "@/modules/user/mock/coupons";
+import { session } from "@/modules/user/mock/session";
+import UserAvatar from "@/modules/user/user/UserAvatar";
+import styled from "@emotion/styled";
+import { CaretRight } from "@phosphor-icons/react";
+import { Button, Card, theme } from "antd";
+import { useRouter } from "next/router";
+
+const OrderCoupon = () => {
+  const {
+    token: { colorPrimary },
+  } = theme.useToken();
+
+  const router = useRouter();
+  const onClickOrderCoupon = () => {
+    void router.push({
+      pathname: "/coupon",
+    });
+  };
+
+  // TODO: get from session
+  const couponInUsed = !!session.coupon;
+  const isHeadTable = session.user?.id === session.headTableUser?.id;
+
+  const coupon = coupons.find((coupon) => coupon.id === session.coupon);
+
+  return (
+    <OrderCouponContainer onClick={onClickOrderCoupon}>
+      <StyledCard>
+        <OrderCouponContent>
+          {!isHeadTable ? (
+            <UserAvatar image={session.headTableUser?.image} />
+          ) : (
+            <UserAvatar image={session.user?.image} />
+          )}
```

Comment:
I'm not sure but could we use just 
`<UserAvatar image={session.headTableUser?.image} />`

Reply:
Done